canonical-web-and-design/ubuntu.com
# Fix link on /smartstart/guide/hardware-setup

## Done

- Fix link on /smartstart/guide/hardware-setup

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/smartstart/guide/hardware-setup
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that 'Canonical’s device enablement' now links to the device enablement page.


## Issue / Card

Fixes #8647
